pkg/oceancd/model/entities.go

```diff
@@ -119,3 +118,102 @@ func ConvertToClusterDetails(cluster map[string]interface{}) ClusterDetails {
 }
 
 //endregion
+
+type VerificationProviderDetails struct {
+	Name       string `header:"Name"`
+	ClusterIDs string `header:"Cluster ID"`
+	Types      string `header:"Type"`
+	UpdatedAt  string `header:"Updated At"`
+}
+
+func ConvertToVerificationProviderDetails(verificationProvider map[string]interface{}) VerificationProviderDetails {
+	clusterIDs := make([]string, 0)
+	types := make([]string, 0)
+
+	verificationProviderName, _ := verificationProvider["name"].(string)
+	updatedAt, _ := verificationProvider["updatedAt"].(string)
+
+	if ids, ok := verificationProvider["clusterIds"].([]interface{}); ok {
```

Comment:
`clusterIds` cannot be null and will always be list of strings, can we change it to 
`clusterIds, _ := verificationProvider["clusterIds"].([]string)`

Reply:
failed to assert, as we get `[]interface{}` there

Reply:
can we at least change this to 
`clusterIds, _ := verificationProvider["clusterIds"].([]interface{})`

Reply:
I see what you mean

Reply:
done